### DISCUSSION TITLE
key to do next list of search

### DISCUSSION THREAD

**Discussion by Fauzanmhr** [mps-youtube/yewtube#1238] [General]
how to do next list on the search ?[
![Screenshot_2023-07-07_08-32-45](https://github.com/mps-youtube/yewtube/assets/105500225/0c420a27-117a-4d2a-a6b1-8f8c96a49263)
](url)

**Comment by Fauzanmhr** [comment]
i was trying to read on help command but still not found :(

**Reply by henkmet** [reply]
try n for next, p for previous.

**Reply by Fauzanmhr** [reply]
thank you very much
